accepted/0000-nohoist.md

```diff
@@ -252,13 +253,22 @@ the file structure will look like this:
 * What other designs have been considered?
 
     - There are a few [high level options](https://github.com/yarnpkg/yarn/issues/3882#issuecomment-338478889) discussed in [#3882](https://github.com/yarnpkg/yarn/issues/3882). 
-    - More implementation alternatives are also marked as discussion in their corresponding sections above.  
-    - we could choose not support nohoist-deep-dep(3), as use case (#2) is not very clear to me if we need this or what the expectation is. However, with inherited nohoistList, we get deep-nohoist pretty much for free so decided to just leave it in. We could either not document it publicly or leave it as it is and refine later when needed.
+    - Some implementation alternatives are discussed inline above. 
+    - The rest is documented in the following section.
 
 # Unresolved questions
 
-## discussion-1
-**is allowing nohoist in public package bad?**
+## dependency tree representation

```

Comment:
@bestander @BYK @arcanis would love to hear your thought on this issue...

Reply:
Thanks, I'll check this later today or tomorrow